src/Mod/Sketcher/Gui/SketcherSettings.cpp

```diff
@@ -49,11 +48,6 @@ SketcherSettings::SketcherSettings(QWidget* parent)
     : PreferencePage(parent), ui(new Ui_SketcherSettings)
 {
     ui->setupUi(this);
-    QGridLayout* gridLayout = new QGridLayout(ui->placeholder);
-    gridLayout->setSpacing(0);
-    gridLayout->setContentsMargins(0, 0, 0, 0);
-    form = new SketcherGeneralWidget(ui->placeholder);
```

Comment:
The class SketcherSettings still has a private class member "form" that isn't used any more.

Reply:
FTR 94d89e53c285c90c7c11dcfa4fd58f0011f63f4c